source/utilities.tex

```diff
@@ -16519,6 +16519,8 @@
   template<class Base, class Derived> struct is_base_of;
   template<class From, class To> struct is_convertible;
   template<class From, class To> struct is_nothrow_convertible;
+  template<class T, class U> struct is_layout_compatible;
+  template<class Base, class Derived> struct is_pointer_interconvertible_base_of;
```

Comment:
This is missing from the paper:
```
template<class Base, class Derived>
inline constexpr bool is_pointer_interconvertible_base_of_v
= is_pointer_interconvertible_base_of<Base,Derived>::value;
```

Reply:
It's further down.

Reply:
Right, the paper placed it incorrectly. I fixed that.